src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,59 @@
+.calendar {
+  display: flex;
+  gap: 1px;
+  padding: 10px;
+  flex-wrap: wrap;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: $cell-width;
+    height: $cell-height;
+    border: 1px solid #000;
+    background-color: #eee;
+
+    font-family: "Arial", sans-serif;
```

Comment:
Set `font-family` for `html` selector. Also you can move `font-size` there, if you have one size in your page